dag/dag.py

```diff
@@ -53,7 +54,8 @@ def size(self):
         return self._size
 
     @classmethod
-    def create(cls, data, links=None, hash_algorithm='sha2-256', serializer=None):
+    def create(cls, data, links=None, hash_algorithm='sha2-256',
```

Comment:
I would set the max line length to 120 chars if that is fine. If it isn't for multiline constructs I prefer doing something like:
```python
def create(
    cls, 
    data, 
    links=None, 
    hash_algorithm='sha2-256', 
    serializer=None
):
```
so that its readable

Reply:
Fine by me, 80 chars is just a habit of mine. Do you happen to know a tidy way to integrate `setup.cfg` flake8 entries with vim?